_rules/SC1-1-1-image-accessible-name-is-placeholder-text.md

```diff
@@ -0,0 +1,69 @@
+---
+name: Image accessibile name is placeholder text
+description: |
+	This rule checks for images that use generic placeholder text.
+
+success_criterion:
+- 1.1.1 # Non-text Content
+
+test_aspects
+- DOM Tree
+- CSS Styling
+
+authors:
+- Bryn Anderson
+---
+
+### Applicability
+
+The rule applies to any HTML `img` element that is exposed to assistive technologies, where the element has a non-empty `alt` attribute that is equal to the accessible name of the element.
+
```

Comment:
I don't find any trace in the previous conversations, but does this rule shouldn't also apply to ìnput type"image"`  and `area` ?

Reply:
Also, why is this limited to 'alt' attribute only? An authoring tool could also produce placeholder 'aria-label' names.

Reply:
"exposed to assistive technologies", "non-empty", and "accessible name" need to be linked to their definitions.

Reply:
It's still about `input type="image"` and `area`.
You rewrite : "with the semantic role of `img`", but  : 
- `input type="image"` has a role `button` so it won't be tested.
- and `area` has a role `link`, so it won't be tested  (https://www.w3.org/TR/html/semantics-embedded-content.html#the-area-element)

But for those two elements, accessible name defined in the alt text, so both can fail (or pass) that test.

I don't really know how to manage this, but I think either we have to create two other tests to handle those two html element (input and area), or we have to rewritte the applicability, but now (and I'm sorry!) I can't find a proposal to make.

Reply:
@audreymaniez I do think separate rules would make more sense for those cases.

Reply:
@kasperisager I don't think either :-)

Maybe it has already been discussed but I can'f find, but can we consider that issue and #263  sort of the same rule ? I mean, maybe we can focus only on "one" and adapt then to the other ? Both issue has relevant reviews for both issues ??

Reply:
@audreymaniez #263 is definitely similar the difference at the moment is 
that this "placeholder text" rule uses the definition of #262 (generic placeholder text) that provides a baseline example for implimenters to follow - in other words placeholder text could be different in French or English so the exact match is an implementation detail.
SC1-1-1-accessible-name-is-image-file-name #263 simply looks for two attributes that match.

Reply:
@audreymaniez In regards to `input type="image"` and `area` can you confirm that you do or do not think they should be applicable in this rule?

I do not think they should be applicable because they do not have the semantic role of `img` and this rule targets accessible names of image elements and not the accessible names of `link` or a `button` elements.

Reply:
I tend to agree with expanding the scope beyond 'img' alone, to include 'input type="image"' and 'area'. I think the rest of the rule works well with that change, and we do not need to have additional rules that do the same thing.

Reply:
Kasper, Audrey and Bryn also agree that this rule should include HTML `input type="image"` and `area`